redaxo/src/core/console.php

```diff
@@ -15,7 +15,9 @@
     }
 }
 
-$application = new rex_console_application();
+rex_console_application::setFactoryClass(rex_console_application_debug::class);
+
+$application = rex_console_application::factory();
```

Comment:
das ist die Quick-And-Dirty Lösung aktuell, da wir in der `boot.php` des Debug-AddOns zu spät zum registrieren sind.